src/Sdk/Sdk.test.js

```diff
@@ -90,7 +90,7 @@ describe('Sdk', () => {
             });
             const result = await prezlySdk.coverage.getByExternalReferenceId(externalReferenceId);
 
-            const search = `?query=%7B%22external_reference_id%22%3A%7B%22%24in%22%3A%5B%22${externalReferenceId}%22%5D%7D%7D`;
+            const search = `?include_deleted=on&query=%7B%22external_reference_id%22%3A%7B%22%24in%22%3A%5B%22${externalReferenceId}%22%5D%7D%7D`;
```

Comment:
A side note: we're sort of lucky that these tests pass.
```
?query=%7B%22external_reference_id%22%3A%7B%22%24in%22%3A%5B%22${externalReferenceId}%22%5D%7D%7D&include_deleted=on
```

would also be a valid value. Running this test in a different environment could theoretically fail. But we can skip it for now though.